detox/android/detox/src/main/java/com/wix/detox/DetoxIdlePolicyConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.wix.detox;
+
+import java.util.concurrent.TimeUnit;
+
+import androidx.test.espresso.IdlingPolicies;
+
+/**
+ * Specification of values to use for Espresso's {@link IdlingPolicies} timeouts.
+ * <br/>Overrides Espresso's defaults as they tend to be too short (e.g. when running on heavy-load app
+ * on suboptimal CI machines).
+ */
+public class DetoxIdlePolicyConfig {
```

Comment:
Extracted from `Detox`, alongside the new `apply()` method so as to move the responsibility where it belongs